client/lib/upgrades/actions/domain-management.js

```diff
@@ -26,10 +26,9 @@ import { isBeingProcessed } from 'lib/domains/dns';
 
 const debug = debugFactory( 'actions:domain-management' );
 
-const sites = sitesFactory(),
-	wpcom = wp.undocumented();
+const wpcom = wp.undocumented();
 
-function setPrimaryDomain( siteId, domainName, onComplete = noop ) {
+const setPrimaryDomain = ( siteId, domainName, onComplete = noop ) => dispatch => {
```

Comment:
Just wondering, why have you opted here for using a callback interface rather than promise interface like in [here](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/actions.js#L105), do we have a "standard" for actions?